async/kafka/kafka.go

```diff
@@ -39,6 +38,19 @@ func init() {
 	prometheus.MustRegister(topicPartitionOffsetDiff)
 }
 
+// ConsumerConfig is the common configuration of patron kafka consumers
```

Comment:
Always end comments with a period
```suggestion
// ConsumerConfig is the common configuration of patron kafka consumers.
```